roles/upgrade/pre-upgrade/tasks/main.yml

```diff
@@ -1,4 +1,19 @@
 ---
+# Wait for upgrade
+- name: Confirm node upgrade
+  pause:
+    echo: yes
+    prompt: "Ready to upgrade node ?"
+  when:
+    - upgrade_node_confirm
+
+- name: Wait before upgrade node
+  pause:
+    seconds: "{{ upgrade_node_pause_seconds }}"
+  when:
+    - not upgrade_node_confirm
+    - upgrade_node_pause_seconds != 0
```

Comment:
upgrade_node_confirm is a bit useless

Reply:
@champtar Is there some other way to better control rolling updates to allow the workload to recover during the draining process.

Reply:
Would be nice to have this included in the next release if possible.